finetune.py

```diff
@@ -267,6 +270,13 @@ def finetune(model, train_loader, train_hiddens, args, device, val_loader=None,
         default=3,
         help="Terminate finetuning if loss doesn't improve after this number of epochs.",
     )
+    parser.add_argument(
```

Comment:
sanity check: should we change this in README or should we keep it at default?

Reply:
I'll update README